### PR TITLE
system: restrict PHP further

### PR DESCRIPTION
**Pull request by fichtner**
Dipping the toe in the php.ini best practices...

* max execution time is recommended as 300 at most via coverity
* We don't handle multiple `$_FILES` except maybe in external backup providers who could create multiple type=file fields.
* having fopen() not read URLs is probably sane enough

### CODE REVIEW

File: src/etc/rc.subr.d/recover [hunk 1]
```diff
@@ -199,6 +200,7 @@ variables_order = "GPCS"
 file_uploads = On
 upload_tmp_dir = /tmp
 upload_max_filesize = 200M
+max_file_uploads = 1
```
Comment (AdSchellevis) [inline src/etc/rc.subr.d/recover:203]: for some plugins this might be challenging, in our code this shouldn't matter.

Comment (fichtner) [reply src/etc/rc.subr.d/recover:203]: I couldn't find a `$_FILES` use in plugins. The default is 20, assuming 200 MB per file I'd say more than 5 is already pretty exhaustive

**Comment by swhite2** [comment]
can't see any weirdness that would be restricted by this, LGTM

File: src/etc/rc.subr.d/recover (hunk 1 above)
Comment (fichtner) [reply src/etc/rc.subr.d/recover:203]: @muratbalaban43 could you check this? Are you using file uploads extensively? Maybe in Phalcon using `getUploadedFiles()`

Comment (fichtner) [inline src/etc/rc.subr.d/recover:203]:
```suggestion
max_file_uploads = 5
```

File: src/opnsense/service/templates/OPNsense/WebGui/php.ini [hunk 2]
```diff
@@ -14,6 +15,7 @@ variables_order = "GPCS"
 file_uploads = On
 upload_tmp_dir = /tmp
 upload_max_filesize = 200M
+max_file_uploads = 1
```
Comment (fichtner) [inline src/opnsense/service/templates/OPNsense/WebGui/php.ini:18]:
```suggestion
max_file_uploads = 5
```

File: src/etc/rc.subr.d/recover (hunk 1 above)
Comment (fichtner) [reply src/etc/rc.subr.d/recover]: @muratbalaban43 going with 5 for now, probably not before 24.7

Comment (muratbalaban43) [reply src/etc/rc.subr.d/recover]: @fichtner thanks for the heads-up. looking into this now.

**Pull request merged**